packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -127,7 +127,7 @@ const Page: NextPageWithLayout = () => {
     { loading: loadingRemoteEntityType },
   ] = useEntityTypeValue(
     baseEntityTypeUri,
-    routeNamespace?.accountId ?? null,
+    routeNamespace?.accountId,
```

Comment:
I think I only changed most of these because they all stemmed from a single type deep in the dependencies, which I had removed for consistency. Now that I've looked at the BP more, I _think_ we're using `null`, so I'll probably change all the types again, but make the consistency `| null` instead of `| undefined`. 

Definitely agree about not drive-by changing these sorts of things randomly though, if it had been random 👍

Reply:
That's been done now